en-US/win10/ArduinoWiringPortingGuide.md

```diff
@@ -0,0 +1,356 @@
+---
+layout: default
+title: Project Setup
+permalink: /en-US/win10/ArduinoWiringPortingGuide.htm
+lang: en-US
+---
+
+# Arduino Wiring Porting Guide
+
+Arduino Wiring sketches and libraries can be copy/pasted into an Arduino Wiring project inside Visual Studio and ran on Raspberry Pi 2 or Minnowboard Max. Sometimes there are slight modifications that need to be made to these files in order to make them more compatable with the Windows environment, or the board you are working with. This guide will cover those modifications as well as common issues that you may run into when deploying Arduino Wiring projects!
+
+### Missing Something?
+Need more information on [Setting up Arduino Wiring in Visual Studio]({{site.baseurl}}/{{page.lang}}/win10/ArduinoWiringProjectGuide.htm)?
+
+## Index
+
+### Porting
+- <a href="#port_pins">Updating Pins</a>
+- <a href="#port_serial">Remove References to "Serial"</a>
+	
+### Common Problems
+- <a href="#prob_missingiot">Can't find "Arduino Wiring Application" Visual C++ project template in Visual Studio</a>
+- ERROR: <a href="prob_hardwareserial">unresolved external symbol "class HardwareSerial Serial"</a>
+- ERROR: <a href="#prob_identifier">"identifier not found" when calling a function</a>
+- <a href="#prob_hanging">My solution hangs infinitely when being initialized</a>
+
+
+
+## Porting
+
+<a name="port_pins"></a>
+
+### Updating Pins
+
+It might go without saying, but many sketches and libraries (especially those for arduino shields) may contain references to specific connector pins for Arduino devices. You'll want to customize your sketches to use the appropriate connector pins for the device you are working on and the configuration you are using.
+
+Arduino Wiring ultimately requires a physical connector pin number for any functions that refer to 'pins'. You can use these numbers directly, but we've also provided some pre-defined pin names which correspond to connector pins on specific boards.
+
+For example, the physical connector pin 29 on a Raspberry Pi 2 is also known as `GPIO_5`. You may set GPIO pin 5 to a HIGH state on a Raspberry Pi 2 by using either of the following commands:
+
+{% highlight C++ %}
+
+pinMode( 29, OUTPUT );
+digitalWrite( 29, HIGH );
+
+{% endhighlight %}
+
+or
+
+{% highlight C++ %}
+
+pinMode( GPIO_5, OUTPUT );
+digitalWrite( GPIO_5, HIGH );
+
+{% endhighlight %}
+
+The pre-defined pin names can be found in `PinNumbers.h` inside any Arduino Wiring project, but since there will be different physical connector pins available depending on the hardware setup you are building for, we've also included a table here to describe which pin names are available for each device.
+
+#### Raspberry Pi 2
+
+[Pinout Diagram]({{site.baseurl}}/images/PinMappings/RP2_Pinout.png)
```

Comment:
I guess that's true, although it can be a little painful to switch between pages to compare pin numbers.
That said, if the image is large enough that adding it still doesn't let you see the table and the image at the same time (i.e. you have to scroll a back and forth), it makes sense to just have the link.